### PR TITLE
Bugfix 0.9.12.2

### PR DESCRIPTION
**Pull request by sserita**
This PR contains three minor bugfixes slated for 0.9.12.2:

1. Tutorial updates for #317 
2. A revamp of the leakage example notebook following broken tests from PR #418
3. Additional labels to indicate report figure/table titles can be clicked for #416 

The leakage notebook may not be in a completely finished state. These updates are more or less making it so that circuit line labels are explicit, but it's a little awkward/still not working completely, especially for the kite structure example at the bottom. Suggestion now are welcome, or we can just slate it reworking later.

### CODE REVIEW

**Comment by coreyostrove** [comment]
Piggybacking on this PR with a bugfix for some reporting code I broke with PR #415 which got caught by the extra tests on beta.

File: jupyter_notebooks/Examples/Leakage.ipynb
Comment (coreyostrove) [inline jupyter_notebooks/Examples/Leakage.ipynb:1]: These changes all look good to me. Great work.

File: pygsti/algorithms/fiducialselection.py
Comment (coreyostrove) [inline pygsti/algorithms/fiducialselection.py:1]: Thanks for implementing this fix!

File: pygsti/report/templates/offline/pygsti_dashboard.js
Comment (coreyostrove) [inline pygsti/report/templates/offline/pygsti_dashboard.js:1]: Great work! This is where I admit to having spent around 45 minutes today trying to implement something similar, but failing on account of quite literally never having written a line of javascript once in my life. Glad to know you're familiar with javascript coding.

**Review by coreyostrove — APPROVED** [review]
Great work! Everything looks good to go to me.

**Comment by coreyostrove** [comment]
Everything looks good to me for the changes that you had made. There is a substantive changed I pushed piggybacking onto this PR. Whenever you get a chance if you could review that (and assuming there aren't any issues therein) then I think this is good to go.

File: pygsti/report/templates/offline/pygsti_dashboard.js
Comment (sserita) [reply pygsti/report/templates/offline/pygsti_dashboard.js:1]: Really had to dust off my JS/HTML/CSS for this, it was my first coding job back when I was a tween 😅

**Review by sserita — COMMENTED** [review]
Your color box plot changes look good!

**Pull request merged**